pants.toml

```diff
@@ -53,6 +53,11 @@ root_patterns = [
   "/build-support/migration-support"
 ]
 
+[python-setup]
+# TODO: Set up an actual lockfile. For now we just "lock" our requirements.txt to itself,
+#  so we can use --python-setup-resolve-all-constraints (which requires a lockfile).
+requirement_constraints = "3rdparty/python/requirements.txt"
```

Comment:
The setting isn't on by default, so I think that this option is not really being used right now.

Reply:
You can turn it on at the cmd line and get much faster tests!

Reply:
It's unclear who would do that though. Putting it in an RC file would be dangerous because you'd forget and not have dependencies enforced, and typing it out each time isn't really feasible. Perhaps you might suggest that it should be on locally and disabled in CI? Unclear.

As mentioned in the body of the review, I think that this option is hard to use correctly, but that there is something else we could do that would be safer.